src/components/Common/HelpCard.tsx

```diff
@@ -1,7 +1,15 @@
 import React from "react";
 import { Card, Row, Col } from "react-bootstrap";
 
-export default function HelpCard({ title, content, icon, theme, link }) {
+type Props = {
+  title: string;
+  content: string;
+  icon: string;
```

Comment:
could be a union type that allows only defined icons.
`type Icon = "computer" | "connection"` (or whatever the icons are called)